rest/src/main/java/fi/otavanopisto/pyramus/rest/controller/permissions/StudentContactLogEntryPermissions.java

```diff
@@ -13,23 +13,23 @@
 public class StudentContactLogEntryPermissions extends AbstractPyramusPermissionCollection implements PyramusPermissionCollection {
```

Comment:
This entire class becomes obsolete if switching to inline checks in the REST service.